library/src/com/google/maps/android/BiMultiMap.java

```diff
@@ -16,40 +17,64 @@
  */
 public class BiMultiMap<K> extends HashMap<K, Object> {
```

Comment:
Class is still public - can you make it package private? I think you'll need to move it but that's probably fine yeah?